lms/djangoapps/course_home_api/dates/v1/tests/test_views.py

```diff
@@ -28,15 +28,10 @@ def test_get_authenticated_enrolled_user(self, enrollment_mode):
         response = self.client.get(self.url)
         self.assertEqual(response.status_code, 200)
 
-        # Pulling out the date blocks to check learner has access. The Verification Deadline Date
-        # should not be accessible to the audit learner, but accessible to the verified learner.
+        # Pulling out the date blocks to check learner has access.
         date_blocks = response.data.get('course_date_blocks')
-        if enrollment_mode == CourseMode.AUDIT:
-            self.assertFalse(response.data.get('learner_is_verified'))
-            self.assertTrue(any(block.get('learner_has_access') is False for block in date_blocks))
-        else:
-            self.assertTrue(response.data.get('learner_is_verified'))
-            self.assertTrue(all(block.get('learner_has_access') for block in date_blocks))
+        self.assertEqual(response.data.get('learner_is_verified'), enrollment_mode == CourseMode.VERIFIED)
+        self.assertTrue(all(block.get('learner_has_access') for block in date_blocks))
```

Comment:
This PR does make one actual change: the VerificationDeadlineDate no longer shows up on the dates tab for audit learners. But I think the fact that it did was an accident in the first place.

It's is_enabled marks it as only for verified learners. So it correctly wasn't showing up in the dates sidebar. But because we didn't separate out those concepts before and never called is_enabled on the dates tab, this was showing up.

It seems like very not useful (and not even tempting?) information to display to an audit learner (this is the paperwork-y you-must-submit-verification-by-this-deadline-to-get-a-cert date, not the upgrade-to-verified-status deadline).